src/QMCWaveFunctions/BsplineFactory/SplineC2ROMPTarget.cpp

```diff
@@ -807,24 +801,37 @@ void SplineC2ROMPTarget<ST>::mw_evaluateVGL(const RefVectorWithLeader<SPOSet>& s
                                             const RefVector<ValueVector_t>& d2psi_v_list) const
 {
   assert(this == &sa_list.getLeader());
-  auto& phi_leader   = sa_list.getCastedLeader<SplineC2ROMPTarget<ST>>();
-  const int nwalkers = sa_list.size();
-  phi_leader.multi_pos_copy.resize(nwalkers * 6);
+  auto& phi_leader = sa_list.getCastedLeader<SplineC2ROMPTarget<ST>>();
+  // make this class unit tests friendly without the need of setup resources.
+  if (!phi_leader.mw_mem_)
+  {
+    app_warning() << "SplineC2ROMPTarget : This message should not be seen in production (performance bug) runs but "
```

Comment:
maybe an #ifndef or NDEBUG assert to avoid the conditional?

Reply:
This is not a stopper but should allow continue running.